proof/crefine/RISCV64/Retype_C.thy

```diff
@@ -1611,16 +1616,18 @@ proof (intro impI allI)
   apply (clarsimp simp: hrs_htd_update ptr_retyps_htd_safe_neg szo
                         kernel_data_refs_domain_eq_rotate
                         ht_rl foldr_upd_app_if [folded data_map_insert_def]
-                        rl projectKOs cvariable_array_ptr_retyps[OF szo]
+                        rl cvariable_array_ptr_retyps[OF szo]
                         zero_ranges_ptr_retyps
               simp del: endpoint_C_size)
+  (* FIXME RT: refill_buffer_relation *)
+  subgoal sorry
```

Comment:
If you put the `FIXME` into the same line as the `sorry`, `git grep sorry` will give a nice overview of what kinds of sorries are left to do.